.ci/ci-pre-checks.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Idea: Fold the logic inside that bash script into the go binary so we don't have to carry around anything else than checkcommits if/when we add this to other repos.